packages/expo-modules-core/react-native.config.js

```diff
@@ -1,6 +1,12 @@
+const path = require('path');
+
 module.exports = {
   dependency: {
     platforms: {
+      ios: {
+        podspecPath: path.join(__dirname, 'ios/ExpoModulesCore.podspec'),
+        project: 'ios/ExpoModulesCore.xcodeproj',
+      },
```

Comment:
This makes `expo-modules-core` linkable by `react-native` CLI. Don't ask me why `podspecPath` must be absolute whereas `project` must be relative 👀 This `*.xcodeproj` doesn't really exist, but passing something here is required to make it linkable 🤷‍♂️

Reply:
i guess this is something that will change when moving to expo package